lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -96,6 +101,7 @@ export class BrowserCacheManager extends CacheManager {
             this.cacheConfig.temporaryCacheLocation,
             this.cacheConfig.cacheLocation
         );
+        this.performanceClient = performanceClient;
```

Comment:
Can we pass this into the function rather than adding it to the instance? It will make de-classing this easier later.

Reply:
done